zcash_primitives/Cargo.toml

```diff
@@ -64,7 +64,7 @@ hdwallet = { workspace = true, optional = true }
 secp256k1 = { workspace = true, optional = true }
 
 # - ZIP 339
-bip0039 = { version = "0.10", features = ["std", "all-languages"] }
+bip0039 = { version = "0.12", features = ["std", "all-languages"] }
```

Comment:
We attempted to upgrade from `bip0039 0.10` in #808, but had to downgrade in #812 because the change in `bip0039 0.11` to use generics resulted in me spending several hours fighting and losing to the C++ template system in `zcashd`. While `zcashd` depends on `bip0039 0.10` via `zcash_primitives::zip339`, we will not be upgrading this dependency. (If someone else wants to fight the C++ template system to enable this, let me know!)